README.md

```diff
@@ -0,0 +1,205 @@
+# storybook-builder-wds
+
+[Storybook builder](https://storybook.js.org/docs/web-components/builders/overview) powered by [@web/dev-server](https://modern-web.dev/docs/dev-server/overview/).
+
+# Install
+
+Install npm packages:
+
+```bash
+npm install @web/dev-server storybook-builder-wds --save-dev
+```
+
+## Setup
+
+Follow the [official storybook docs](https://storybook.js.org/) to setup storybook.
+
+You can choose Vite when asked for the builder in the init CLI.
+Later you can change it to `storybook-builder-wds` in the storybook main configuration file.
+
+If you are migrating from the `@web/dev-server-storybook` plugin, please read the [migration guide](#migration) below.
+
+## Configuration
+
+When you finish the setup you'll have a standard storybook folder with configuration files (`.storybook` by default) containing at least `main.js`.
+
+### Activating the builder
+
+You'll need to activate `storybook-builder-wds` manually.
+To do that change the following in `main.js`:
+
+- set `config` type to `import('storybook-builder-wds').StorybookConfigWds`
+- set the `core.builder` setting to `'storybook-builder-wds'`
+- keep the `framework.name` (Vite won't be used as a builder, but the chosen framework, e.g. `web-components`, will be inherited from this setting)
+
+```js
+// .storybook/main.js
+
+/** @type { import('storybook-builder-wds').StorybookConfigWds } */
+const config = {
+  ...
+  framework: {
+    name: '@storybook/web-components-vite',
```

Comment:
Should we just create a framework for this? A "framework" is only an object with a `builder` property and a `renderer`, e.g.:

```js
export const core = {
  builder: 'storybook-builder-wds',
  renderer: '@storybook/web-components',
};
```

Reply:
there will be a framework indeed, I was planning to refine it tomorrow